topics/imaging/tutorials/electrophoresis-gel-bands-image-analysis/tutorial.md

```diff
@@ -0,0 +1,322 @@
+
+Reliable measurement of band intensity provides quantitative insights into gene expression and other biochemical properties ({% cite Gassmann2009 %}). However, manual quantification can be labor-intensive and prone to user bias.
+
+Modern image analysis platforms such as **QuPath** ({% cite Bankhead2017 %}) provide interactive tools for defining **regions of interest (ROIs)** directly on gel images. These ROIs can be exported as **GeoJSON** files, which encode the spatial coordinates of the selected bands in a standardized, machine-readable format.
+
+Once the ROI file is generated, bioinformatics platforms like **Galaxy** ({% cite Galaxy2024 %}) enable automated processing of these coordinates, measurement of pixel intensities for each band, and generation of transparent, reproducible quantification results.
+
+Recent advances such as **GelGenie** ({% cite Aquilina2025 %}) further highlight the trend towards AI-powered, automated analysis pipelines for electrophoresis gels, reinforcing the need for accessible, reproducible workflows that integrate modern image processing tools.
+
+
+> <agenda-title></agenda-title>
+>
+> The steps covered in this tutorial are:
+>
+> 1. Uploading and preparing the electrophoresis gel image  
+> 2. Selecting and exporting regions of interest (ROIs) using QuPath  
+> 3. Converting ROI coordinates to a label map  
+> 4. Quantifying band intensity  
+> 5. Inspecting results
+>
+{: .agenda}
+
+# Data preparation
+
+The example dataset can be downloaded from here [Electrophoresis gel dataset](https://github.com/galaxyproject/training-material/tree/main/topics/imaging/images/electrophoresis-gel-bands-image-analysis/Image.jpg).
+
+![Electrophoresis Gel](../../images/electrophoresis-gel-bands-image-analysis/Image.jpg)
+
+This image was published by [Kenji Ohgane et al. (2019) on Protocols.io](https://www.protocols.io/view/quantification-of-gel-bands-by-an-image-j-macro-ba-bp2l6n4bkgqe/v1).
+
+> <hands-on-title>Download and upload the image</hands-on-title>
+>
+> 1. Create a new history in Galaxy.  
+> 2. Download the test image using the link above.  
+> 3. Click the **{% icon galaxy-upload %} Upload** to add the image file into your galaxy history.  
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
+>
+> 4. Rename {% icon galaxy-pencil %} the dataset to `Electrophoresis-Gel.jpg`.
+>
+{: .hands_on}
+
+Before we dive into the quantification steps let's convert our jpg image file into the image format TIFF. This format is preferred for analysis and quantification as it preserves the image data and metadata. 
```

Comment:
Please use consistent capitalization for file formats, like `jpg` _and_ `tiff` or `JPG` _and_ `TIFF`:

```suggestion
Before we dive into the quantification steps let's convert our JPG image file into the TIFF image format. This format is preferred for analysis and quantification as it preserves the image data and metadata. 
```